cypress/support/commands.ts

```diff
@@ -18,6 +19,28 @@ declare global {
 
 Cypress.Commands.add("mount", mount);
 
+const terminalLog = (violations: Result[]) => {
+  cy.task(
+    "log",
+    `${violations.length} accessibility violation${
+      violations.length === 1 ? "" : "s"
+    } ${violations.length === 1 ? "was" : "were"} detected`
+  );
+  // pluck specific keys to keep the table readable
+  const violationData = violations.map(
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
Taking out this ignore comment doesn't cause the compiler to throw a type error, so it looks like we should be safe to remove it.

```suggestion
```